src/snippets/glamorous.js

```diff
@@ -127,35 +127,27 @@ const SubmitButton = glamorous.button(
     marginTop: '1rem'
   },
   props => ({
-    backgroundColor: props.disabled ? '#BBB' : undefined
+    backgroundColor: props.disabled ? '#BBB' : null
   })
-);
-
-SubmitButton.defaultProps = {
-  type: 'submit'
-};
+).withProps({type: 'submit'});
```

Comment:
This is the only substantive change and it's really not that big. It would be faster to just use the `defaultProps` but the `withProps` API composes really well so it's nice to stick with that :)

Reply:
I actually like the API you've shown here too. I think I picked up `defaultProps` from the Glamorous documentation, so I'll try and find that and maybe show the `withProps` instead.

Reply:
Thanks! The `withProps` is relatively new so I wouldn't be surprised if `defaultProps` is in the docs.